presto-native-execution/presto_cpp/main/types/PrestoTaskId.h

```diff
@@ -61,10 +66,14 @@ class PrestoTaskId {
   }
 
  private:
-  int nextDot(const std::string& taskId, int start) {
+  int nextDot(const std::string& taskId, int start, bool allowEnd) {
```

Comment:
Let's add TODO() to not to forget to remove that compatibility code.
cc @shrinidhijoshi

Reply:
@spershin I added a TODO here
https://github.com/prestodb/presto/pull/19739/files#diff-c77fd18228c604e811fd4458782b93fac38a2d901be0ab60b3ac7a88d4663121R26